integration_tests/src/main/python/delta_lake_low_shuffle_merge_test.py

```diff
@@ -0,0 +1,211 @@
+# Copyright (c) 2023-2024, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pyspark.sql.functions as f
+import pytest
+import string
+
+from asserts import *
+from data_gen import *
+from delta_lake_utils import *
+from marks import *
+from pyspark.sql.types import *
+from spark_session import *
+
+# Databricks changes the number of files being written, so we cannot compare logs
+num_slices_to_test = [10] if is_databricks_runtime() else [1, 10]
+
+delta_merge_enabled_conf = copy_and_update(delta_writes_enabled_conf,
+                                           {
+                            "spark.rapids.sql.command.MergeIntoCommand": "true",
+                            "spark.rapids.sql.command.MergeIntoCommandEdge": "true",
+                            "spark.rapids.sql.delta.lowShuffleMerge.enabled": "true"
+                                            })
```

Comment:
Nit: The indentation is really weird here, can't figure out what rule it's trying to follow.